checkoff

```diff
@@ -1,167 +1,162 @@
 #!/usr/bin/env python3
 import argparse
+import getpass
 import json
-import os
 import subprocess
 import sys
 
 from ocflib.account.search import user_exists
+from ocflib.infra.db import get_connection
 
+MYSQL_PASSWORD_FILE = 'mysql_pwd'
```

Comment:
feels dirty, alternatives welcome

Reply:
maybe we can put this in `/opt/mysql_pwd` on new decal bastion or something? follows the ocf scheme